src/Compatibility/Core/src/iOS/Renderers/PageRenderer.cs

```diff
@@ -135,9 +135,6 @@ public override void ViewDidLayoutSubviews()
 
 			if (_safeAreasSet || !Forms.IsiOS11OrNewer)
 				UpdateUseSafeArea();
-
-			if (Element.Background != null && !Element.Background.IsEmpty)
-				NativeView?.UpdateBackgroundLayer();
```

Comment:
Can you rebase and just remove the changes from the Compatibility project?

We could even move versions of UpdateBackgroundLayerFrame into compatibility
Once we delete them from controls

We try to never touch any of the code inside `Compatibility`